Add functionality to find-feature-flags script to find defines that are only used once

mokjasmine opened the pull request:
#### f0a1be091b779a7d1477127cea731e1764a51779
<pre>
Add functionality to find-feature-flags script to find defines that are only used once
<a href="https://bugs.webkit.org/show_bug.cgi?id=252763">https://bugs.webkit.org/show_bug.cgi?id=252763</a>

Reviewed by NOBODY (OOPS!).

It might be useful to know which defines are only used in one file. Knowing such may help us clean up the code and remove any underutilized defines. This PR adds the functionality of printing out single use defines to the find-feature-flags script.

-Tools/Scripts/find-feature-flags
-Tools/Scripts/webkitpy/featuredefines/search.py
</pre>
----------------------------------------------------------------------
#### ac7c707b38a221fa23b2af33c71ea6356aac854a
<pre>
Add functionality to find-feature-flags script to find defines that are only used once
<a href="https://bugs.webkit.org/show_bug.cgi?id=252763">https://bugs.webkit.org/show_bug.cgi?id=252763</a>

Reviewed by NOBODY (OOPS!).

It might be useful to know which defines are only used in one file. Knowing such may help us clean up the code and remove any underutilized defines. This PR adds the functionality of printing out single use defines to the find-feature-flags script.

-Tools/Scripts/find-feature-flags
-Tools/Scripts/webkitpy/featuredefines/search.py
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f0a1be091b779a7d1477127cea731e1764a51779

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/110090 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/19192 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42763 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/1526 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/119106 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/114042 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/20654 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/10383 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/102349 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115836 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/15376 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98584 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43597 "Passed tests") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/5/builds/113529 "Passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/97337 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/30244 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/85429 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11889 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31582 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/12508 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8544 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17873 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/51197 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/14307 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

emw-apple reviewed (commented):
Looks good to me and seems useful. If you'd like to take this further, I think it'd be better to **count** the number of times a definition is used, instead of special-casing for definitions that are only used once. Knowing that a definition is only used twice is just as useful!

`grep -c` is good precedent to replicate here. We could have a `-c|--counts` flag that would output a feature macro with the number of times it's used, e.g.

```
ENABLE_2022_GLIB_API:1
ENABLE_3D_TRANSFORMS:2
ENABLE_ACCESSIBILITY:3
ENABLE_ACCESSIBILITY_ANIMATION_CONTROL:4
…
```

mokjasmine committed:
Add functionality to find-feature-flags script to find defines that are only used once
https://bugs.webkit.org/show_bug.cgi?id=252763

Reviewed by NOBODY (OOPS!).

It might be useful to know which defines are only used in one file. Knowing such may help us clean up the code and remove any underutilized defines. This PR adds the functionality of printing out single use defines to the find-feature-flags script.

-Tools/Scripts/find-feature-flags
-Tools/Scripts/webkitpy/featuredefines/search.py

mokjasmine committed:
Add functionality to find-feature-flags script to find defines that are only used once
https://bugs.webkit.org/show_bug.cgi?id=252763

Reviewed by NOBODY (OOPS!).

It might be useful to know which defines are only used in one file. Knowing such may help us clean up the code and remove any underutilized defines. This PR adds the functionality of printing out single use defines to the find-feature-flags script.

-Tools/Scripts/find-feature-flags
-Tools/Scripts/webkitpy/featuredefines/search.py
